usb/Makefile

```diff
@@ -10,6 +10,9 @@ LOCAL_HEADERS := hcd.h hub.h dev.h drv.h usbhost.h
 LIBS := $(USB_HCD_LIBS) $(USB_HOSTDRV_LIBS) libusb
 DEPS := libusb
 
+CFLAGS += -flto
+LDFLAGS += -flto
```

Comment:
These should be `LOCAL_CFLAGS` and `LOCAL_LDFLAGS` - you want to enable LTO only for this specific component and not others